src/system/AdminModule/Controller/AdminInterfaceController.php

```diff
@@ -0,0 +1,773 @@
+    					$menuitem['icon'] = 'eye';
+    				} elseif ($menuitem['class'] == 'z-icon-es-warning') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'exclamation-triangle';
+    				} elseif ($menuitem['class'] == 'z-icon-es-rss') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'rss-square';
+    				}
+    			}
+    	
+    			$active = '';
+    			if (!empty($menuitem['url']) && System::getBaseUrl().$menuitem['url'] === System::getCurrentUrl()) {
+    				$active = 'active ';
+    			}
+    	
+    			$dropdown = '';
+    			if (isset($menuitem['links'])) {
+    				$dropdown = 'dropdown' ;
+    			}
+    	
+    			$html .= '<li';
+    			$html .= !empty($menuitem['id']) ? ' id="'.$menuitem['id'].'"' : '';
+    			$html .= ' class="'.$active.$dropdown;
+    			$html .= !empty($class) ? $class : '';
+    			$html .= '">';
+    			$attr  = !empty($menuitem['title']) ? ' title="'.$menuitem['title'].'"' : '';
+    			$attr .= !empty($menuitem['class']) ? ' class="'.$menuitem['class'].'"' : '';
+    	
+    			if (isset($menuitem['disabled']) && $menuitem['disabled'] == true) {
+    				$html .= '<a '.$attr.'>'.$menuitem['text'].'</a>';
+    			} elseif (!empty($menuitem['url'])) {
+    				$icon = '';
+    				if (!empty($menuitem['icon'])) {
+    					$icon = '<span class="fa fa-'.$menuitem['icon'].'"></span> ';
+    				}
+    				$html .= '<a href="'.DataUtil::formatForDisplay($menuitem['url']).'"'.$attr.' style="display: inline-block;">'.$icon.$menuitem['text'].'</a>';
+    				if (isset($menuitem['links'])) {
+    					$html .= '<a href="#" class="dropdown-toggle" data-toggle="dropdown" style="text-decoration: none;">&nbsp;<b class="caret"></b></a>';
+    				}
+    			} else {
+    				$html .= '<span'.$attr.'>'.$menuitem['text'].'</span>';
+    			}
+    			if (isset($menuitem['links'])) {
+    				$html .= '<ul class="dropdown-menu">';
+    				foreach($menuitem['links'] as $submenuitem) {
+    					$html .= '<li>';
+    					if (isset($submenuitem['url'])) {
+    						$html .= '<a href="'.DataUtil::formatForDisplay($submenuitem['url']).'">'.$submenuitem['text'].'</a>';
+    					} else {
+    						$html .= $submenuitem['text'];
+    					}
+    					$html .= '</li>';
+    				}
+    				$html .= '</ul>';
+    			}
+    			$html .= '</li>';
+    		}
+    	
+    		$html .= '</ul>';
+    	}    	
+    	
+    	return ['menu' => $html];
+    }    
+    
+    
+    /**
+     * Get security analyzer data.
+     *
+     * @return array data
+     */
+    private function _securityanalyzer()
+    {
+        $data = array();
+
+        // check for magic_quotes
+        $data['magic_quotes_gpc'] = DataUtil::getBooleanIniValue('magic_quotes_gpc');
+
+        // check for register_globals
+        $data['register_globals'] = DataUtil::getBooleanIniValue('register_globals');
+
+        // check for config.php being writable
+        $data['config_php'] = (bool)is_writable('config/config.php');
```

Comment:
this really doesn't matter anymore. maybe it can be left, but the installer automatically makes the file not writable and - now that I think of it, the file in question is actually `personal_config.php` now. but the symfony config files are the ones that matter. this file will eventually be eliminated.